src/cr/cube/cube.py

```diff
@@ -547,19 +547,30 @@ def _slice_idxs(self):
             return (0,)
         return range(len(self.dimensions[0].valid_elements))
 
+    @lazyproperty
+    def _show_missing(self):
+        """True if cube dimensions doesn't contain MR and NUM_ARRAY else False."""
+        return not all(d in self.dimension_types for d in [DT.MR, DT.NUM_ARRAY])
```

Comment:
This would be easier to understand if you put explicit return paths for concrete types (there's only two of them, so no harm done), something like:
```
if DT.MR in self.dimension_types:
    return False
if DT.NUM_ARRAY in self.dimension_types:
    return False
return True

Reply:
this is not the same condition. I have to return false only if all the dimension types matched
In your example you return false right after an MR is in the dimension types that is not correct. This rule has the goal of aviod show missing summary row only if the cube is MRxNUMARR and vice versa